ui-admin/src/portal/siteContent/designer/editors/StyleEditor.tsx

```diff
@@ -62,6 +85,25 @@ export const StyleEditor = ({ section, updateSection }: {
                   updateSection({ ...section, sectionConfig: JSON.stringify({ ...config, blurbAlign: opt?.value }) })
                 }}/>
             </div>}
+        {Object.hasOwnProperty.call(config, 'blurb') && <div>
+          <label className="form-label fw-semibold">Blurb Text Size</label>
```

Comment:
use the `useReactSingleSelect` helper, especially using the `selectInputId` returned -- that will let you make the label accessible